src/node_contextify.cc

```diff
@@ -246,14 +246,22 @@ class ContextifyContext {
     // there is no way for the embedder to tell if the data index is
     // in use.
     struct ScopedEnvironment {
+      bool alreadyInDebugContext = false;
```

Comment:
I rebased in those style changes in case we decide to move forward with 803ca9667a8dc608c5271633e42e792dbba66fc0